docs/website/pages/index.vue

```diff
@@ -64,17 +63,17 @@
       </div>
     </div>
 
-    <div class="w-3/4 mx-auto mb-20">
+    <div class="w-3/4 mx-auto mb-20 pt-4">
       <div class="flex flex-col justify-center">
         <div class="text-center">
-          <h2>API Driven</h2>
+          <h2>API Control</h2>
```

Comment:
I don't know if "API Control" sounds good. I don't like "API Driven" neither though.

Reply:
Hmm...what about one of these:

"Modern CLI and API"
"API and CLI Driven"
"Automation-Ready CLI and API"

Something like that...it occurs to me that we are talking about an API and then showing a CLI, and maybe we should be explicit about it? Not sure!

Reply:
I don't know the best title. We could drop it really and just have the terminal. We could then add a feature bullet point around the API.

Reply:
Yeah, that's a good idea.